src/components/modals/DokanModal.tsx

```
@@ -0,0 +1,157 @@
+
+                    { dialogContent || (
+                        <div className="sm:flex sm:items-start min-h-32">
+                            {dialogIcon || (
+                                <div
+                                    className={`flex items-center justify-center flex-shrink-0 w-14 h-14 bg-red-50 border border-red-50 rounded-full`}>
+                                    <DialogIcon />
+                                </div>
+                            )}
+                            <div className="mt-3 text-center sm:ml-4 sm:mt-0 sm:text-left">
+                                <h3 className="text-base font-semibold text-gray-900"
+                                    dangerouslySetInnerHTML={
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)